specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/scheduledActions.json

```diff
@@ -0,0 +1,1008 @@
+          "format": "int32"
+        },
+        "daysOfWeek": {
+          "description": "Day names in english on which cost analysis data will be emailed. This property is applicable when frequency is Weekly or Monthly.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DaysOfWeek"
+          }
+        },
+        "weeksOfMonth": {
+          "description": "Weeks in which cost analysis data will be emailed. This property is applicable when frequency is Monthly and used in combination with daysOfWeek.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/WeeksOfMonth"
+          }
+        },
+        "dayOfMonth": {
+          "type": "integer",
+          "description": "UTC day on which cost analysis data will be emailed. Must be between 1 and 31. This property is applicable when frequency is Monthly and overrides weeksOfMonth or daysOfWeek.",
+          "format": "int32"
+        },
+        "startDate": {
+          "description": "The start date and time of the scheduled action (UTC).",
+          "type": "string",
+          "format": "date-time"
+        },
+        "endDate": {
+          "description": "The end date and time of the scheduled action (UTC).",
+          "type": "string",
+          "format": "date-time"
+        }
+      }
+    },
+    "systemData": {
+      "description": "Metadata pertaining to creation and last modification of the resource.",
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "createdBy": {
+          "type": "string",
+          "description": "The identity that created the resource."
+        },
+        "createdByType": {
+          "type": "string",
+          "description": "The type of identity that created the resource.",
+          "enum": [
+            "User",
+            "Application",
+            "ManagedIdentity",
+            "Key"
+          ],
+          "x-ms-enum": {
+            "name": "createdByType",
+            "modelAsString": true
+          }
+        },
+        "createdAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The timestamp of resource creation (UTC)."
+        },
+        "lastModifiedBy": {
+          "type": "string",
+          "description": "The identity that last modified the resource."
+        },
+        "lastModifiedByType": {
+          "type": "string",
+          "description": "The type of identity that last modified the resource.",
+          "enum": [
+            "User",
+            "Application",
+            "ManagedIdentity",
+            "Key"
+          ],
+          "x-ms-enum": {
+            "name": "createdByType",
+            "modelAsString": true
+          }
+        },
+        "lastModifiedAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The timestamp of resource last modification (UTC)"
+        }
+      }
```

Comment:
Instead can reference

[https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/types.json#L528](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/types.json#L528)

and/or

[https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/types.json#Definitions/ProxyResource](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/types.json#Definitions/ProxyResource),


and/or

[https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/types.json#Definitions/TrackedResource](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v3/types.json#Definitions/TrackedResource),

Reply:
Reused the one from v3/types.json